lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -1627,6 +1647,20 @@ void IRGenDebugInfoImpl::popLoc() {
   std::tie(LastDebugLoc, LastScope) = LocationStack.pop_back_val();
 }
 
+/// This is done for WinDbg to avoid having two non-contiguous sets of
+/// instructions because the ``@llvm.trap`` instruction gets placed at the end
+/// of the function.
+void IRGenDebugInfoImpl::setInlinedTrapLocation(IRBuilder &Builder,
+                                                const SILDebugScope *Scope) {
+  if (Opts.DebugInfoFormat != IRGenDebugInfoFormat::CodeView)
```

Comment:
I think so. It would needlessly increase the size of the debug info and be confusing to users if it were not the arithmetic operator that did the trapping.